ui/apps/dashboard/src/components/RunDetails/useGetTraceResult.ts

```diff
@@ -1,35 +1,29 @@
 import { useCallback } from 'react';
-import type { Result } from '@inngest/components/types/functionRun';
 import { useClient } from 'urql';
 
 import { useEnvironment } from '@/components/Environments/environment-context';
 import { graphql } from '@/gql';
 
 const query = graphql(`
-  query TraceResult($envID: ID!, $traceID: String!) {
+  query TraceResult($envID: ID!, $runID: String!) {
     workspace(id: $envID) {
-      runTraceSpanOutputByID(outputID: $traceID) {
-        data
-        error {
-          message
-          name
-          stack
-        }
+      run(runID: $runID) {
```

Comment:
**question**: does this endpoint also use Clickhouse as the source of truth or is this an old endpoint that uses Big Query? I want to make sure that it uses the same source of truth as the rest of the new UI

Reply:
It's ClickHouse. This is the same field that the run details queries:
https://github.com/inngest/inngest/blob/aaron/inn-3422-execute-run-trace-and-output-queries-in-parallel-not/ui/apps/dashboard/src/components/RunDetails/useGetRun.ts#L46